gdc_maf_tool/cli.py

```diff
@@ -79,3 +80,18 @@ def main() -> None:
 
     with open(args.output_filename, "wb") as f:
         aggregate_mafs(mafs, f)
+
+        failed_uuids = [m.file.uuid for m in mafs if m.file.failed]
```

Comment:
Because of the delayed download feature we have to collected failed downloads after the aggregation takes place. Only after then will the tool attempt to download the file. After that we know if it fails.